src/components/InputPassword/InputPassword.css

```diff
@@ -0,0 +1,14 @@
+.Input-Password{
```

Comment:
Please implement with Pascal Case
```suggestion
.InputPassword{
```

Reply:
@Mubangizi this has been fixed.